community/gegl/APKBUILD

```diff
@@ -27,12 +27,12 @@ build() {
 
 check() {
 	cd "$builddir"
-	make check
+	#make check
```

Comment:
If for whatever reasons tests can't be run, it's preferable that are disabled via "options=" and adding a comments why.
Thanks!

Reply:
Comitted accidentally, reverted now.